### PR TITLE
PUT PageLoader

### PR DESCRIPTION
**Pull request by patgrudniewski**

### CODE REVIEW

File: src/page/UpdateForm.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { compose } from 'redux';
+import { reduxForm } from 'redux-form';
+
+import { CreateForm } from './CreateForm.js';
```
Comment (auamarto) [inline src/page/UpdateForm.js:6]: .js is needed ?

Comment (patgrudniewski) [reply src/page/UpdateForm.js]: [here](https://github.com/opstalent/redux-crud/pull/18/files#diff-d8d6d6933ff93f5a4ca3064abb323ad5R43)

Comment (patgrudniewski) [reply src/page/UpdateForm.js:6]: it is kind of a pattern I'm using here to avoid disambiguation, because there are also `md` files

Comment (patgrudniewski) [reply src/page/UpdateForm.js:6]: it is not needed but it is used in all files across lib

Comment (auamarto) [reply src/page/UpdateForm.js:6]: ok

**Review by auamarto — APPROVED** [review]

**Review by Druzzdil — APPROVED** [review]

**Pull request merged**